osm2gtfs/creators/ci_abidjan/trips_creator_ci_abidjan.py

```diff
@@ -0,0 +1,209 @@
+            service_days = self._DAYS_OF_WEEK
+
+        if collections.Counter(service_days) == collections.Counter(self._DAYS_OF_WEEK):
+            return self._date_range('monday', 'sunday')
+        if collections.Counter(service_days) == collections.Counter(self._DAYS_OF_WEEK[:5]):
+            return self._date_range('monday', 'friday')
+        if collections.Counter(service_days) == collections.Counter(self._DAYS_OF_WEEK[:6]):
+            return self._date_range('monday', 'saturday')
+        if collections.Counter(service_days) == collections.Counter(self._DAYS_OF_WEEK[-2:]):
+            return self._date_range('saturday', 'sunday')
+
+        return ','.join([self._DAY_ABBREVIATIONS[day_name] for day_name in service_days])
+
+    def _set_default_times(self, transport_hour):
+        transport_hour['start_time'] = self._DEFAULT_START_TIME
+        transport_hour['end_time'] = self._DEFAULT_END_TIME
+
+    def _init_service_period(self, feed, hour):
+        service_id = self._days_abbreviation_from_transport_hour(hour)
+        service_period = ServicePeriod(id=service_id)
+        service_period.SetStartDate(self.config['feed_info']['start_date'])
+        service_period.SetEndDate(self.config['feed_info']['end_date'])
+        for i,day_of_week in enumerate(self._DAYS_OF_WEEK):
+            if (day_of_week in hour and hour[day_of_week]):
+                service_period.SetDayOfWeekHasService(i)
+        feed.AddServicePeriodObject(service_period)
+        print("Created new service period " + service_id)
+        return service_period
+
+    def _process_transport_hours(self, feed, transport_hours_list):
+        transport_hours_dict = {}
+        for hour in transport_hours_list:
+            service_id = self._days_abbreviation_from_transport_hour(hour)
+            try:
+                feed.GetServicePeriod(service_id)
+            except KeyError:
+                self._init_service_period(feed, hour)
+
+            if "00:00:00" ==  hour['start_time'] and hour['end_time'] in ["00:00:00", "24:00:00"]:
+                self._set_default_times(hour)
+
+            if service_id in transport_hours_dict.keys():
+                transport_hours_dict[service_id].append(hour)
+            else:
+                transport_hours_dict[service_id] = [hour]
+        return transport_hours_dict
+
+    def _init_default_hour(self):
+        hour = { key:True for key in self._DAYS_OF_WEEK }
+        self._set_default_times(hour)
+        hour['headway'] = self._DEFAULT_HEADWAY * 60
+        return hour
+
+    def _init_agency(self, feed, agency_name):
+        return feed.AddAgency(agency_name, None, self.config['agency']['agency_timezone'], agency_id=agency_name)
+
+    def add_trips_to_feed(self, feed, data):
+        lines = data.routes
+        default_hour = self._init_default_hour()
+        default_service_period = self._init_service_period(feed, default_hour)
+        default_hours_dict = {
```

Comment:
It would be better to init the default service period just like the others: 
```
transport_hours = transporthours.main.Main()
default_hour = transport_hours.tagsToGtfs({
        "opening_hours": "Mo-Su,PH 05:00-22:00",
        "interval": "00:30",
        })
default_hours_dict = self._process_transport_hours(feed, default_hour)
```

Reply:
ok